app/edge/services/edge-group-service.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
```

Comment:
File should be renamed as `edge-group`, it's already in the service layer (`service` folder) no need for that extra redundancy.